code/modules/food_and_drinks/drinks/drinks.dm

```diff
@@ -402,11 +402,12 @@
 	isGlass = FALSE
 
 /obj/item/reagent_containers/food/drinks/soda_cans/suicide_act(mob/living/carbon/user)
-	user.visible_message("<span class='suicide'>[user] is trying to eat \the [src]! It looks like [user.p_theyre()] trying to commit suicide!</span>")
-	var/obj/item/trash/can/crushed_can = new /obj/item/trash/can(user.loc)
-	crushed_can.icon_state = icon_state
-	qdel(src)
-	return BRUTELOSS
+	user.visible_message("<span class='suicide'>[user] takes a big sip! It looks like [user.p_theyre()] trying to commit suicide!</span>")
+	playsound(user.loc,'sound/items/drink.ogg', 80, 1)
+	reagents.clear_reagents() //a big sip
+	user.say(pick("Now, Outbomb Cuban Pete, THAT was a game.", "They don't make 'em like Orion Trail anymore.", "You know what they say. Worst day of spess carp fishing is better than the best day at work.", "They don't make 'em like Singularity engines anymore.'"))
```

Comment:
`They don't make 'em like Singularity engines anymore.'`
bad apostrophe and capitalization

Reply:
English capitalization rules are the devil